keyboards/null/st110r2/readme.md

```diff
@@ -0,0 +1,31 @@
+# null/st110r2
+
+null ST110r2 is a custom PCB/keyboard project that supports multiple layouts and is built on top of the Filco and the clones alike.
+
+![st110r2_zoom](https://s3.amazonaws.com/null-src/images/posts/nkbm-st110r2/NKBM-ST110r21-both1.png)
+![st110r2_full](https://s3.amazonaws.com/null-src/images/posts/nkbm-st110r2/NKBM-ST110r21-both3.png)
+
+* Keyboard Maintainer: [user176176](https://github.com/user176176)
+* Hardware Supported: [null] 最高タイプ (SaikouType) NKBM-ST110r2.x keyboard PCBs with Teensy 2.0++.
+* Hardware Availability: [NKBM-ST110r2 - null-src.com](https://null-src.com/posts/nkbm-st110r2/post.php)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make null/st110r2:default
```

Comment:
```suggestion
    qmk compile -kb null/st110r2 -km default
```
is the new way

Reply:
This recommendation is currently not required.

Reply:
I actually used the CLI / qmk compile and qmk flash but the qmk new-keyboard had these as defaults so I didn't dare to change the "official way of doing this".